packages/ses/src/tame-faux-data-properties.js

```diff
@@ -0,0 +1,124 @@
+import {
+  getOwnPropertyDescriptor,
+  apply,
+  defineProperty,
+  toStringTagSymbol,
+} from './commons.js';
+
+/**
+ * Exported for convenience of unit testing. Harmless, but not expected
+ * to be useful by itself.
+ *
+ * @param {any} obj
+ * @param {string|symbol} prop
+ * @param {any} expectedValue
+ * @returns {boolean}
+ */
+export const tameFauxDataProperty = (obj, prop, expectedValue) => {
+  if (obj === undefined) {
+    return false;
+  }
+  const desc = getOwnPropertyDescriptor(obj, prop);
+  if (!desc || 'value' in desc) {
+    return false;
+  }
+  const { get, set } = desc;
```

Comment:
Perhaps a comment: “Here we attempt to discover whether the getter and setter behave like a value property, modulo side-effects we cannot observe and do not care about.”

Reply:
I added a bunch of comments that I think makes this point well enough with other words. Done.